### ISSUE TITLE
Adding new lines after custom BBCodes (in WYSIWYG) editor

### ISSUE THREAD

**Issue by cyphix333** [samclarke/SCEditor#517]
I have created some custom bbcodes and one of them is a block level element of which I have this set:

My BBCode object:

``` javascript
var my_bbcode =
{
    tags: {
        "span": {
            "class": ["my_bbcode"]
        }
    },
    isInline: false,
    breakAfter: true,
    format: '[mytag]{0}[/mytag]',
    html: '<span class="my_bbcode">{0}</span>',
};
```

However the `breakAfter` doesn't appear to work. Whenever I insert this tag you cannot get your cursor on the next line since it is a block level element.

**Comment by cyphix333** [comment]
Ok, it seems it works fine (with our without the `breakAfter: true` if you highlight the data in _WYSIWYG _ mode and use the toolbar button, but if you do it in _source_ mode then switch back to _WYSIWYG _ mode then you can't go down a line without specifically adding one manually in _source_ mode.

**Comment by brunoais** [comment]
Did you try instead of:

	isInline: false,

use:

	isInline: true,

?

**Comment by cyphix333** [comment]
Tried that but no change.

**Comment by samclarke** [comment]
Thanks for reporting! Fixed by (39750149d7228cad541b74878e5d45a1a1aa3237).
